CMakeLists.txt

```diff
@@ -6,6 +6,8 @@ set(PACKAGE_NAME matplotlib_cpp)
 set(INSTALL_CONFIGDIR ${CMAKE_INSTALL_LIBDIR}/${PACKAGE_NAME}/cmake)
 
 
+option(BUILD_EXAMPLES "Build examples" OFF)
```

Comment:
should prepend options with `MATPLOTLIB_CPP_*`, .e.g., `MATPLOTLIB_CPP_BUILD_EXAMPLES`